src/app/[locale]/globals.css

```
@@ -22,126 +22,243 @@
-	--aura-hue: 20deg;
-	--spacing: 0.38rem;
-	--surface-rgb: 253, 246, 240;
-	--selector-txt: #22222c;
-	/*Brands*/
-	--nextjs-fill: #171717;
-	--html-fill: #000;
-	--github-fill: #0f0e0fff;
-	text-rendering: optimizeLegibility;
-	-webkit-font-smoothing: antialiased;
-	-moz-osx-font-smoothing: grayscale;
+  /*LightTheme*/
```

Stylelint (17.6.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)